Dockerfile

```diff
@@ -175,6 +175,7 @@ COPY --link --from=buildctl /usr/bin/buildctl /
 COPY --link --from=buildkitd /usr/bin/buildkitd /
 
 FROM scratch AS binaries-darwin
+COPY --link --from=buildkitd /usr/bin/buildkitd /
```

Comment:
We do not want to ship stub buildkitd binary for macos users.

Reply:
Okay, will remove from here. Do you want me to add another job that would check that it compiles?

Reply:
You can add it to this matrix: https://github.com/moby/buildkit/blob/ea6f91e09ab21d391a1d909daf8cceaa198a0a37/.github/workflows/test-os.yml#L33-L35

```yaml
      matrix:
        platform:
          - windows/amd64
          - freebsd/amd64
          - darwin/amd64
```

Reply:
If I read things properly, that job builds goes through the same Dockerfile target (`binaries`), so it won't build `buildkitd`.

Reply:
Hum right it will build `binaries-for-test` target: https://github.com/moby/buildkit/blob/ea6f91e09ab21d391a1d909daf8cceaa198a0a37/.github/workflows/test-os.yml#L57

https://github.com/moby/buildkit/blob/ea6f91e09ab21d391a1d909daf8cceaa198a0a37/docker-bake.hcl#L102-L106

https://github.com/moby/buildkit/blob/ea6f91e09ab21d391a1d909daf8cceaa198a0a37/Dockerfile#L367-L371

We would just want `buildctl` and `buildkitd`

Reply:
Soo... Do you want me to add `COPY --link --from=buildkitd /usr/bin/buildkitd /` to `binaries-for-test`? It will do redundant work for other platforms, but I don't see any other good way.

Reply:
I guess we can create a dedicated job in `test-os` workflow to build `buildkitd` target for `darwin/amd64` platform. I can push these changes to your branch if you want.

Reply:
> I can push these changes to your branch if you want.

That would be cool!